frontend/packages/integration-tests-cypress/cypress.json

```diff
@@ -0,0 +1,13 @@
+{
+  "integrationFolder": "tests",
+  "screenshotsFolder": "../../gui_test_screenshots/cypress/screenshots",
+  "videosFolder": "../../gui_test_screenshots/cypress/videos",
```

Comment:
This doesn't seem to match what's in gitignore above.

Reply:
removed those lines from `.gitignore`.